pkg/platform/allocator/vacate.go

```diff
@@ -174,6 +174,8 @@ func waitVacateCompletion(p *pool.Pool, hasWork bool) error {
 	return errs.ErrorOrNil()
 }
 
+// nolint better to have this grouped rather than scattered around, there's
```

Comment:
Maybe I haven't got enough ☕️ yet but I don't fully understand this comment here 😊